### PR TITLE
Implement new landing design

### PR DESCRIPTION
**Pull request by zoek1**
##### Description
Implement the new landing design for desktop and mobile.

##### Refers/Fixes
No issue related yet

##### Testing
https://github.com/gitcoinco/web/pull/6468

image

### CODE REVIEW

File: app/assets/v2/css/home.css [hunk 1]
```diff
@@ -2,6 +2,113 @@
...
+
+.dark-section {
+  background-color: #170640;
+  color: white;
+}
+
+.section-title {
+  color: #121315;
+  font-size: 2.5rem;
+  display: inline-block;
```
Comment (thelostone-mc) [inline app/assets/v2/css/home.css:24]: ^ let's use bootstrap classes

File: app/assets/v2/css/home.css [hunk 2]
```diff
@@ -2,6 +2,113 @@
...
+
+.dark-section .section-title {
+  color: white;
+  border-bottom: 0.68rem solid white;
+}
+
+.how-it-works img {
+  width: 75%;
+  margin-left: auto;
+  margin-right: auto;
```
Comment (thelostone-mc) [inline app/assets/v2/css/home.css:36]: ^ let's use bootstrap classes

File: app/assets/v2/css/home.css [hunk 3]
```diff
@@ -2,6 +2,113 @@
...
+}
+.testimonial-card {
+  margin-top: 24px;
+  border: 4px solid #3E00FF;
+  display: inline-block;
```
Comment (thelostone-mc) [inline app/assets/v2/css/home.css:41]: ^ let's use bootstrap classes

File: app/assets/v2/css/home.css [hunk 4]
```diff
@@ -2,6 +2,113 @@
...
+.testimonial-card:first-child {
+  margin-left: -25%;
+}
+.testimonial-card:last-child {
+  margin-right: -25%;
+}
+.card-description {
+  width: 60%;
+  margin-left: auto;
+  margin-right: auto;
```
Comment (thelostone-mc) [inline app/assets/v2/css/home.css:54]: ^ let's use bootstrap classes `mx-auto`

File: app/assets/v2/css/home.css [hunk 5]
```diff
@@ -2,6 +2,113 @@
...
+  font-size: 28px;
+  line-height: 36px;
+  text-align: center;
```
Comment (thelostone-mc) [inline app/assets/v2/css/home.css:57]: ^ let's use bootstrap classes `text-center`

File: app/assets/v2/css/home.css (hunk 2 above)
Comment (thelostone-mc) [inline app/assets/v2/css/home.css:36]: ^ let's use boostrap

File: app/assets/v2/css/home.css [hunk 6]
```diff
@@ -23,6 +130,7 @@
 }
 
 .banner__h2 {
+  font-size: 2.4rem;
```
Comment (thelostone-mc) [inline app/assets/v2/css/home.css:133]: won't the existing font-size classes work ?

File: app/assets/v2/css/home.css [hunk 7]
```diff
@@ -319,6 +427,13 @@ figure:hover::before {
   animation: rock_arrow ease-in-out 4s infinite;
 }
 
+.hide_on_mobile {
+  display: block;
+}
+.show_on_mobile {
+  display: none;
+}
```
Comment (thelostone-mc) [inline app/assets/v2/css/home.css:435]: ^ let's use bootstrap classes
both of those can we replaced with `d-none d-sm-block` sort of thing

File: app/assets/v2/css/home.css [hunk 8]
```diff
@@ -444,3 +559,111 @@ figure:hover::before {
...
+  }
+  .avatar {
+    max-width: 60px;
+    max-height: 60px;
+  }
+  .press-mobile {
+    padding-left: 35px;
+    padding-right: 35px;
+  }
+}
```
Comment (thelostone-mc) [inline app/assets/v2/css/home.css:669]: ^ Add an empty  line after every rule declaration 
Let's reuse use font classes we already have + bootstrap classes

File: app/retail/views.py [hunk 9]
```diff
@@ -1654,51 +1613,120 @@ def tribes(request):
...
-            'img': static('v2/images/tribes/landing/stats.svg'),
+            'img': static('v2/images/report.svg'),
             'info': 'See how your hackathons are performing'
+        },
+        {
+            'title': 'Kudos',
+            'img': static('v2/images/kudos.svg'),
+            'info': 'Show appreciation to your tribe members'
+        }
+    ]
```
Comment (thelostone-mc) [inline app/retail/views.py:1659]: ^ why are these files added again ?
Let's reuse what we have + remove duplicate svgs and let's keep them under `'v2/images/tribes/landing`
to keep it a better organized

File: app/retail/views.py [hunk 10]
```diff
@@ -1654,51 +1613,120 @@ def tribes(request):
...
+        },
+        {
+            'link': 'https://www.ethnews.com/gitcoin-offers-bounties-for-ens-integration-into-dapps',
+            'img': 'v2/images/press/ethnews.jpg'
+        },
+        {
+            'link': 'https://www.hostingadvice.com/blog/grow-open-source-projects-with-gitcoin/',
+            'img': 'v2/images/press/hosting-advice.png'
         }
     ]
```
Comment (thelostone-mc) [inline app/retail/views.py:1698]: ^ This is duplicated information. Move it to separate file and import it. aka DRY code

File: app/retail/views.py [hunk 11]
```diff
@@ -1654,51 +1613,120 @@ def tribes(request):
...
+            'alt': 'commit reveal scheme'
+        },
+        {
+            'link': 'https://medium.com/gitcoin/announcing-open-kudos-e437450f7802',
+            'img': static("v2/images/medium/3.png"),
+            'title': _('Announcing Open Kudos'),
+            'description': _('Our vision for integrating Kudos in any (d)App'),
+            'alt': 'open kudos'
+        }
+    ]
```
Comment (thelostone-mc) [inline app/retail/views.py:1722]: ^ This is duplicated information. Move it to separate file and import it. aka DRY code

File: app/retail/views.py [hunk 12]
```diff
@@ -1654,51 +1613,120 @@ def tribes(request):
...
     context = {
-        'plans': plans,
         'tribes': tribes,
         'testimonials': testimonials,
-        'reasons': reasons
+        'reasons': reasons,
+        'articles': articles,
+        'press': press,
+        'top_tribes': top_tribes
```
Comment (thelostone-mc) [inline app/retail/views.py:1729]: tribes already shows the top tribes ! Why do we have a duplicate one ?

image

**Comment by zoek1** [comment]
Thanks @thelostone-mc, I'll implement the changes and I'll ping you when is ready!

File: app/assets/v2/css/home.css (hunk 6 above)
Comment (zoek1) [reply app/assets/v2/css/home.css:133]: No, the font-size in the mockup is different than the existing font-size classes

File: app/assets/v2/css/home.css (hunk 7 above)
Comment (zoek1) [reply app/assets/v2/css/home.css:435]: That was my first approach but for some reason doesn't have any effect on the images so I implemented in this way.

**Comment by zoek1** [comment]
Ok, this is ready :)

File: app/assets/v2/css/home.css [hunk 13]
```diff
@@ -2,6 +2,107 @@
...
+.tribe-points p {
+  font-size: 1.5rem;
+}
+.tribe-points li {
+  padding-left: 30px;
+  padding-right: 30px;
+}
+.tribes {
+  background-color: #EFF3F6 !important;
+}
```
Comment (thelostone-mc) [inline app/assets/v2/css/home.css:105]: Could we add in empty lines after every css rule?

Comment (thelostone-mc) [reply app/assets/v2/css/home.css:117]: nvm fixed but do keep this in mind the next time @zoek1 ^_^

**Comment by octavioamu** [comment]
@PixelantDesign said is not ready so getting back to Draft, feel free to change when ready @zoek1

**Comment by thelostone-mc** [comment]
@zoek1 when do you those above changes -> lets removing the svg/jpg which we won't be using

**Comment by frankchen07** [comment]
some small comments

Screen Shot 2020-05-04 at 17 37 02

I'm not quite sure people will understand this section. Are all of these things included in the Tribes package? To me it seems a little random.

Screen Shot 2020-05-04 at 17 37 19

Can we remove Gitcoin Labs? It's been gone for a while now.

**Comment by frankchen07** [comment]
is there a way to see the changes since the last few comments?

**Comment by frankchen07** [comment]
last thing is just seeing if we can get the tribes video in the next 12-18 hours, if not we remove from the landing page for now.

File: app/assets/v2/images/tribes/landing/host-workshop.svg [hunk 14]
```diff
@@ -0,0 +1,10 @@
...
...I+ST95RvzyglUekewxbFvZ4g6Q6XGxMpZBsCDRwhVphJbZM/5dolAIhGyXgf9fQZWlHiMlK4nJ9E6EQgp6ehQ+Pv/s3fnLq4VM1VWQ06PnIBo/EUHZAN0COjBoOkGoALAzU7BnMCZ9ix0ew1/xMGPGGGYZ3mlRtpx0FKgeSCndboW3tG1Z5I5JQJq2srsSsqyHDJmlncHc4hd53P1Lpb5M+yd/IyTCLewti6/ezs8RfG2NKrRl+2iqsMtEw/r/6UQBesQSByvS4SbYY37FxCRGaUOgcnUp2ShZgtkNkm2thRP9q4pwbI/NqiidR3803fLo0W708UMWuRTXw0w/k5uA+hGgyW/pyyDkR6HSBu5b+dxNT+97N52lPxqCa0eg5HcH5mFCFYps/+H6dYn4WB91qRlwm+pZas3rdkjbvimsLRIWDlwPgInHHv+sUScme866DcQRVcr1jZFO1UynpXGKm92GysInwx1Zvy8hL+Uq4GJ2vUPM/J6jMxsc2bi9kPN8vK4CdnIzpq8jJOxNEG3FsNEX5FVfUpAlvtkrE1j4gv9x67U1eHh7fQi9ETxKA3QL+ZpLDy9PByaAZDyjm6BJD9dVy/fEm4s6baqbesPCKUemTQawDs+wUn7m3rSW6EXPH0swqN8UOwkwLomJSW779l2VFxGRPYzOzYdEq17Z1Rvh/1MB7KH9PoUtxRvyXHxSBiQiAdWJ/bT0z0ZQiBxuHvVG8WaMbCSB/ec4LnzeDP/OlWtLJ1ou3vp/DXhHYBOuJ0Ju6Q6N+Hg+ziMnaNwwuxcPD2tyIZpY3HvSi4UWc+gqcaAksTgXxK2M0pjSUdDpwMtXZKTX7m3e32Kg7sVRwEc+DYaYTaOXNPXWZJhTl8uB6nJ640WOE7WTxRNcghyG0ENuPb3beFJR/pTp3jaCgJ8uQHGWY9xwiW7VAfVyVZBvvCvw6AZwMDI56zZl7epn+nWCLrE+fyZK6JjTf27LgLHyY+ZS7uXDCMQqpTpUsTVDMzsac5mqDiMVwB20YCEcY9x2v4WQJv0OZ/mHsiLqgXd4TOBgZclP3UMzJGzk3gO/mMEQujqKJeIg0rOUAIxNYX5Qd+dg/VmQE0OcelyOGDcB7Dl5OLK3OScN2d7gKzUPMA7MDC63UnHwKONYCfiEbuShIbO4rXN938vPfg8hOJ8mQAAAABJRU5ErkJggg=="/>
```
Comment (octavioamu) [inline app/assets/v2/images/tribes/landing/host-workshop.svg:8]: seems we have a bitmap image here inside the svg

File: app/assets/v2/images/tribes/landing/grow-tribe.svg [hunk 15]
```diff
@@ -0,0 +1,49 @@
...
...r2q0fLZtlICuzfpqDqKlv3WaCK+WaEnUJFaqr0RUlfLpcmzexIze79tAFakm1a1k/e6y4VkZlZpJlt2CHo4FXRP18qhLANGmf/xAEbtQzKTQHbFYJrVuobS82yrOicX79nHBCbRMST19BVa11qtAlSP23hgGX5Yh/QZP7h0wKzQECiwDTUrtWmbDUQ6tQwMX81MWalbYcqXBfD2xMTLEVcNEuRl1VchKTs9SLNTvLgmaF+gKMDLuEoqbqHi0ZJVCGhUzLckxtlDUZ6SciDcOAS9EntOYk1vBKIioh2XE6nF7s3OeEy+0NSLqk1IWSUvUJLgGeEvBANLkgVToTkELyozQOuKpqKyQrt26KpMlnffAUA5t8cL91ecP9XxTBb9/twKGSGnmjZWW1B1t2OuB2e2GxJDoHgbmmKQIfCbYYfc8aoMMY4Kqu8Rn3bJopVFIDrBBmOXAmFN+CeL3AoaM16Nw+S1483rjdgSPHXChumwFnjRfVjuSDS9zKI1UYo+fSH3jHty4THhutzypEdRqTwnOAANpz0IH2rTJ986+ycjcOlvh5GT6W+PkQN3iaw8L48TX2mFz+OYIpzIicnYePunD8RC3v3G5g5z6/aezIY4mPz4wM/xqbUXouQwwLpZNGaFjNtEthUuQc2LHXgRoXBRzV8pAw8pDx9amy9XHScGt8U0h+bIYYFkKoDBNc0TUiSgw3bq+Gwxm55d7oUojMt7LthL4lHSlmZLGkli9D9FyK+TSyltvHTYqOA/UNrIB6Ez6W0ZUktXwboiVKHr90ywRXajWwSHMj1luGK9JQqe3PEOBKZM8lzgVEd2pXa+JyJ9pIjGcqIrjaVKe+abhIym4Iu4XZry+C/fM1cnlZSVnZWeAMwgUPrJBglc81jYQdgX5q4IGryiEvtFKiJc8NJPmU1EDPmifyepUX4OmSNZJXXr9SnnE1i6IXCyRI/o5XE4P6VikGvXvhlcvIK0mS/2rXyJgG70Xypys+PekWysP4lOzUpqP273W54XI4ZWOjoSSzzKs/HsV1sryM0v9IzrTb44GjqvbYXEumHaVTr1Mn3ADvjCHQEJR2eaROZUWtaTXWRzxHGDylg794UTzzFq88RRoPezDxBMxIw0Xiz1PTkDnjL6EhhoWuAXU/ENzPCtOVKhwwyuF4/w8Cu0UV5afwhwAAAABJRU5ErkJggg=="/>
```
Comment (octavioamu) [inline app/assets/v2/images/tribes/landing/grow-tribe.svg:44]: png (bitmap) in a SVG, need to be a svg or a png

**Comment by owocki** [comment]
such excite for this
